packages/hooks/src/test/index.test.js

```diff
@@ -494,6 +494,11 @@ test( 'Test doingAction, didAction and hasAction.', () => {
 	doAction( 'another.action' );
 	expect( doingAction( 'test.action' ) ).toBe( false );
 
+	expect( didAction( 'unattached.action' ) ).toBe( 0 );
+	doAction( 'unattached.action' );
```

Comment:
If I duplicate this line (i.e. run the action twice before checking count), didAction still returns 1 where I would expect it to return 2. Am I misunderstanding how this is intended to behave?

Reply:
You're absolutely correct, good catch, @kadamwhite!